src/guiChatConsole.cpp

```diff
@@ -661,4 +661,3 @@ void GUIChatConsole::setVisible(bool visible)
 		recalculateConsolePosition();
 	}
 }
-
```

Comment:
Both the C and C++ standards say that there must be a blank line at the end of the file to be compliant with the standard (to see the warning you would need to use -pedantic though). So put it back :P